docs/install.rst

```diff
@@ -35,6 +35,17 @@ pypromice is also provided with a conda environment configuration environment.ym
 .. _GitHub: https://github.com/GEUS-Glaciology-and-Climate/pypromice
 .. _environment.yml: https://github.com/GEUS-Glaciology-and-Climate/pypromice/blob/main/environment.yml
 
+The package has inbuilt unit tests, which can be ran to test the package installation:
```

Comment:
It should be "run" instead of "ran", I think.

Reply:
Done!